.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -83,7 +83,8 @@ Check that the release will work:
 - [ ] Update crate versions, commit the changes to the release branch, and do a release dry-run:
 
 ```sh
-cargo release version --verbose --execute --allow-branch '*' --workspace --exclude zebrad beta
+cargo release version --verbose --execute --allow-branch '*' --workspace --exclude zebrad --exclude zebra-scan beta
+cargo release version --verbose --execute --allow-branch '*' --package zebra-scan alpha # Increment version manually if this doesn't work
```

Comment:
Note: The automated command should work, because the version has to be changed in every crate that depends on `zebra-scan` as well.

Reply:
It works for beta, and the deserialization for the `Alpha` variant looks correct. May be a bug in cargo-release. 

Replaced with a fastmod command in https://github.com/ZcashFoundation/zebra/pull/8027/commits/0f84d526b98916d9144273634c2884fb169aa379. 

Hopefully zebra-scan will be in beta within a couple releases or cargo-release fixes a bug affecting alpha version bumps.